src/components/info/RelationsList.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+	import type { SearchResult } from "src/api/search";
+	import Relation from "./relation/Relation.svelte";
+
+	export let relations: SearchResult[];
+
+	$: relationTypes = relations.reduce((obj, current) => {
+		if (!current.relation) return obj;
+
+		if (current.relation.type in obj)
+			obj[current.relation.type].push(current);
+		else obj[current.relation.type] = [current];
+
+		return obj;
+	}, {} as Record<string, SearchResult[]>);
+</script>
+
+{#each Object.entries(relationTypes) as [type, results]}
+<h2 class="h4 mb-0">Has {type}...</h2>
+
+<ul>
+	{#each results as { content, relation }}
```

Comment:
How easy would it be to separate out the inverse relations? "A worked at  B" is very different from "A employed B" -- and "C mentored D" is even harder to tease out from "D mentored C," but just as different!

Reply:
This wouldn't be too difficult - I'd just need to branch out if `content.id === relation.link_to` in the `.reduce` function here - and create a map of alternate heading names for the inverse relations.

If I should implement this, does this set of inverse headings look okay?
- "Has worked at..." -> "Has employed..."
- "Has mentored..." -> "Has been mentored by..."
- "Has studied at..." -> "Has educated..." (or "Has taught"? "University of Pittsburgh has taught James" doesn't sound quite right to me, but idk)
- "Has worked alongside..." -> "Has worked alongside..."
- "Has served on..." -> (I'm not sure what makes sense here)